integration_tests/mockApis/auth.ts

```diff
@@ -149,14 +153,22 @@ const stubUserRoles = () =>
       headers: {
         'Content-Type': 'application/json;charset=UTF-8',
       },
-      jsonBody: [{ roleCode: 'SOME_USER_ROLE' }],
+      jsonBody: [{ roleId: 'SOME_USER_ROLE' }],
```

Comment:
Looks like it _should_ be `roleCode`? (https://github.com/ministryofjustice/hmpps-template-typescript/commit/a6c72bae0519636b005981a4cc381f522599dae7#diff-80ecc0c5d6a711027b0c3af4ca9715f1a1917775f01ea78b9258226d56fe7172R150)

Reply:
Updated and replaced with roleCode